transcom/mymove
# update-country-code-from-unitedstates-to-us

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16799)

## Summary

The getMoveTaskOrder api for the Prime can return United States or US, but it should only be returning US. This is happening because early in the application's history, data had been inputted with country codes that were more than 2 digits, this has since changes and we want to make sure that the application returns the 2 digit country code all the time. 

This PR updates the addresses table to replace instances of 'United States' with 'US', it's 2 digit country code.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. In the `main` branch check the addresses table, country column, for references to 'United States'
2. Then log into this branch [NT-MB-16799-update-country-code-to-US](https://github.com/transcom/mymove/tree/NT-MB-16799-update-country-code-to-US) and check that the same addresses have an updated country code of 'US'

### Database

#### Any new migrations/schema changes:


## Screenshots
